networking/rosa-shared-vpc-config.adoc

```diff
@@ -0,0 +1,26 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-shared-vpc-config"]
+= Configuring a shared virtual private cloud for ROSA clusters
+:context: rosa-shared-vpc-config
+
+toc::[]
+
+You can share virtual private cloud (VPC) configurations with different AWS accounts within your AWS organization. This process requires two separate AWS accounts in the same AWS organization. One account functions as the VPC owner while the other account creates the cluster in this VPC.
+
+.Prerequisites
+* You have installed the latest ROSA CLI (`rosa`).
```

Comment:
Should we go for `1.2.26`?
If we say the latest, the docs will be outdated once `1.2.27` is out, as `1.2.26` also supported the shared VPC flow.

Reply:
`1.2.26 or above`?